plugin/src/makeWorklet.ts

```diff
@@ -235,6 +237,17 @@ export function makeWorklet(
         )
       )
     );
+    if (!isRelease() && process.env.REANIMATED_PLUGIN_TESTS !== 'jest') {
```

Comment:
Let's move this condition to a separate function like so:
```ts
export function isJest() {
  return process.env.REANIMATED_PLUGIN_TESTS === 'jest';
}
```
Then let's wrap this condition in another function:
```ts
export function shouldInjectWorkletVersion() {
  return !isRelease() && !isJest();
}
```
This way we get more self-descriptive code.